zulipterminal/ui_tools/views.py

```diff
@@ -141,17 +141,21 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
                 return super(MessageView, self).keypress(size, 'page up')
 
         elif is_command_key('SCROLL_TO_BOTTOM', key) and not self.old_loading:
-            if self.focus_position == len(self.log) - 1:
+            if self.focus is not None and \
+               self.focus_position == len(self.log) - 1:
                 return self.keypress(size, 'down')
             else:
                 return super(MessageView, self).keypress(size, 'page down')
 
         elif is_command_key('THUMBS_UP', key):
-            self.model.react_to_message(self.focus.original_widget.message,
-                                        reaction_to_toggle='thumbs_up')
+            if self.focus is not None:
+                self.model.react_to_message(
+                    self.focus.original_widget.message,
+                    reaction_to_toggle='thumbs_up')
 
         elif is_command_key('TOGGLE_STAR_STATUS', key):
-            message = self.focus.original_widget.message
+            if self.focus is not None:
+                message = self.focus.original_widget.message
             self.model.toggle_message_star_status(message)
```

Comment:
I'm not sure if your local branch is up to date with this one, but this doesn't fix the issue. I was alerted by reading this, since message is not defined if `self.focus` is `None`, but this also fails when I try to star the message when running it (as one might expect).

Do you see the problem?

Reply:
@neiljp . I am afraid i don't understand. This seems to work fine in my branch.
The normal error which causes the crash is:
```
File "/home/sumanthvrao/Git_Hub/zulip-terminal/zulipterminal/ui_tools/views.py", line 158, in keypress
    message = self.focus.original_widget.message
AttributeError: 'NoneType' object has no attribute 'original_widget'
```
Doesn't this mean that the message doesn't exist and self.focus object is None?

Reply:
Oh i am sorry! I had somehow not indented the 
`self.model.toggle_message_start_status` line due to which it was failing.`
This seems to work when i tried some manual testing. Is this not the approach desired?